### PR TITLE
chore: Minor typo cleanup

### PR DESCRIPTION
**Pull request by nschonni**
Took a quick pass with cSpell

### CODE REVIEW

File: tests/require-passive-events.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ ruleTester.run('require-passive-events', rule, {
       ]
     },
     {
-      // Intentionally mispelled!
+      // Intentionally misspelled!
```
Comment (srt32) [inline tests/require-passive-events.js:50]: :)

File: docs/rules/array-foreach.md [hunk 2]
```diff
@@ -58,7 +58,7 @@ for (const apple of apples) {
 }
 ```
 
-If `polishApple` needed to do some serial async work, then we'd need to refactor the iteration steps to accomodate for this async work, by `await`ing each call to `polishApple`. We cannot simply pass an `async` function to `forEach`, as it does not understand async functions, instead we'd have to turn the `forEach` into a `reduce` and combine that with a `Promise` returning function. For example:
+If `polishApple` needed to do some serial async work, then we'd need to refactor the iteration steps to accommodate for this async work, by `await`ing each call to `polishApple`. We cannot simply pass an `async` function to `forEach`, as it does not understand async functions, instead we'd have to turn the `forEach` into a `reduce` and combine that with a `Promise` returning function. For example:
 
 ```diff
 - apples.forEach(polishApple)
```
Comment (srt32) [inline docs/rules/array-foreach.md:64]: Could you explain this diff here?

**Review by srt32 — COMMENTED** [review]
Great catches!

File: docs/rules/array-foreach.md (hunk 2 above)
Comment (nschonni) [reply docs/rules/array-foreach.md:64]: The line you're commenting on is a code block with `diff` language tag, not something I changed

File: tests/require-passive-events.js (hunk 1 above)
Comment (nschonni) [reply tests/require-passive-events.js:50]: Yeah, I didn't correct the intentional typo below :)

File: docs/rules/array-foreach.md (hunk 2 above)
Comment (srt32) [reply docs/rules/array-foreach.md:64]: aha! Yes.

**Review by srt32 — APPROVED** [review]

**Pull request merged**